### DISCUSSION TITLE
Грязная машина - нужно помыть | Механика грязи

### DISCUSSION THREAD

**Discussion by IlonRudkovskyi** [octothorp-team/public-dobrograd#798] [Предложения]
**Привет!**
Предлагаю ввести систему загрязнения корпуса авто (можно реализовать как по таймеру так и по определённым зонам и/или погоде), ведь не бывает такого что бы машина постоянно была чистой, верно? "Зачем это нужно?" - ответ прост как пять копеек: для поддержания ролевой атмосферы. 

**Пример:** вы выехали в пригород покатались по просёлочной дороге полной грязи, а следов этой самой грязи как таковой на вашей машине то и нет. Да, вы скажете что вы можете это отыграть эмоутом НО зачем каждый раз кидать эмоут когда это можно реализовать такой механникой, тем более что грязные скины на авто уже есть на сервере.

**Касаемо** загрязнения разобрались, теперь касательно того "а как её отмыть?" - можно добавить рп айтем в магазин механнику ("Моющее" or "Кёрхер" or "Тряпка" (название придумайте сами) ) заюзав которое по автомобилю ( с довольно долгим процессом использования ) будет делать авто чистым!

**Comment by Starikprostoy** [comment]
Согласен с вами, это не только увеличит РП состовляющую, но еще и добавит новый вид заработка-автомойки.

**Comment by Lesopoval** [comment]
Идея крутая, я хочу предложить немного другую, свою версию:
Думаю что лучше добавить возможность мыть машину для всех профессий, во вкладке "Повседневное". И будет оно называться как: "Моющее средство" или как-либо иначе в этом роде. Цена его будет от 500 до 1000х рублей. Просто если это будет только у механика, то эта мойка станет ограниченной, ибо механиков на дбг довольно мало. Ну думаю не стоит объяснять. Это моя версия, надеюсь она будет вовлечена во внимание автора!
